server/index.js

```diff
@@ -1,10 +1,116 @@
 require('dotenv/config');
+const pg = require('pg');
+const argon2 = require('argon2');
 const express = require('express');
-const staticMiddleware = require('./static-middleware');
+const jwt = require('jsonwebtoken');
+const ClientError = require('./client-error');
+const errorMiddleware = require('./error-middleware');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
 
 const app = express();
 
-app.use(staticMiddleware);
```

Comment:
When you deploy you will need the `staticMiddleware` so you should add that back in, in the future.